client/js/upload-handler/upload.handler.controller.js

```diff
@@ -687,6 +686,10 @@ qq.UploadHandlerController = function(o, namespace) {
             handler.reset();
         },
 
+        resetPausedUpload: function() {
```

Comment:
much better! I'll re-run manual tests when i get a chance. not sure i will have time today, but we shall see